lib/item.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Item
+  def initialize(stats)
+    @stats = stats
+  end
+
+  def id
+    @stats[:id]
+  end
+
+  def name
+    @stats[:name]
+  end
+
+  def description
+    @stats[:description]
+  end
+
+  def unit_price
+    BigDecimal(@stats[:unit_price], 4)
+  end
+
+  def created_at
+    @stats[:created_at]
+  end
+
+  def updated_at
+    @stats[:updated_at]
+  end
+
+  def merchant_id
+    @stats[:merchant_id]
+  end
+
+  def unit_price_to_dollars
+    unit_price.round(2, :half_up).to_f
+  end
+
+  def update(attributes)
```

Comment:
I wonder if this could / should be abstracted into smaller methods? I don't necessarily know if there's VALUE in that, but it does seem to function in a way that's updating multiple attributes.